sktime/forecasting/base/_base.py

```diff
@@ -105,6 +105,8 @@ def fit(self, y, X=None, fh=None):
         self._X = X
         self._y = y
 
+        self._set_cutoff(y.index[-1])
```

Comment:
This is something we forgot in #912, right?

Reply:
If so, let's get this merged soon. I has been open for six days already, so I'll merge it later today or tomorrow unless there are any other comments.

Reply:
right, but nothing is failing since the individual forecasters *also* do this currently

Reply:
I also interpret your comment that you'll decide to resolve this convo and/or merge, not me (as the PRer usually would)

Reply:
Isn't this something that happens in `_set_fh` too? I'll check this later.

Reply:
Happy for you to merge, but let me check something first. Usually the reviewer merges no?

Reply:
Looks all good, doesn't happen in `_set_fh`, got confused.

Reply:
? I thought it´s usually the PRer. See e.g., here:
https://docs.github.com/en/get-started/quickstart/github-flow
"Once your pull request is approved, merge your pull request."

The reason, I guess, is that´s it´s a classical handshake protocol - both PRer and reviewer signal the go-ahead, the reviewer by approving, the PRer by merging. If the reviewer merges, the final handshake is missing a hand.

Reply:
anyway, merged